components/sharing/SocialShareModal.vue

```diff
@@ -5,16 +5,46 @@
     hide-header-close
     hide-footer
     data-qa="share modal"
+    v-on="onItemPage && !oEmbedDataHtml?
+      { hide: resetEmbedCopied, show: requestEmbed}
+      :onItemPage ? { hide: resetEmbedCopied }
+        :{}"
   >
     <div class="icon-wrapper">
       <SocialShare
         :media-url="mediaUrl"
         with-text
       />
     </div>
+    <b-form
```

Comment:
This should go into its own component, e.g. /components/item/ItemEmbedCode.vue.

SocialShareModal should then have an unnamed [slot](https://vuejs.org/v2/guide/components-slots.html) at this position, which the ItemHero would use to insert ItemEmbedCode, thus:
```
<SocialShareModal
  :media-url="selectedMedia.about"
>
  <ItemEmbedCode
    :identifier="identifier"
  />
</SocialShareModal>
```